docs/pages/management/export-audit-events/fluentd.mdx

```diff
@@ -228,6 +228,16 @@ accept events using TLS and print them:
       time_type string
       time_format %Y-%m-%dT%H:%M:%S
     </parse>
+
+    <buffer>
```

Comment:
I think it would make sense to add a short comment above this line indicating why this block is here and whether the user is expected to modify it (no, right?).

Reply:
Done in 67c4c377959f39b0bec8af94c1d0351b65762560